android/src/main/java/dev/openfeature/sdk/FlagEvaluationMetadata.kt

```diff
@@ -0,0 +1,63 @@
+package dev.openfeature.sdk
+
+class EvaluationMetadata internal constructor(private val values: Map<String, Any>) {
+
+    fun getString(key: String): String? = values[key] as? String
+
+    fun getBoolean(key: String): Boolean? = values[key] as? Boolean
+
+    fun getInt(key: String): Int? = values[key] as? Int
+
+    fun getDouble(key: String): Double? = values[key] as? Double
+
+    fun getAny(key: String): Any? = values[key]
```

Comment:
Following the [specification](https://openfeature.dev/specification/sections/providers#requirement-2210) I don't think we need this `getAny` in the `EvaluationMetadata`.

Reply:
Ah! I think this is just a remnant from an attempt I made with an operator function (but I failed).
I'll push an update that removes it.